variables.tf

```diff
@@ -16,64 +16,72 @@
 
 variable "organization_id" {
   description = "The organization id for putting the policy"
-  default     = ""
+  type        = number
+  default     = null
 }
 
 variable "folder_id" {
   description = "The folder id for putting the policy"
-  default     = ""
+  type        = number
+  default     = null
 }
 
 variable "project_id" {
   description = "The project id for putting the policy"
-  default     = ""
+  type        = string
+  default     = null
 }
 
 variable "enforce" {
   description = "If boolean constraint, whether the policy is enforced at the root; if list constraint, whether to deny all (true) or allow all"
-  default     = ""
+  type        = bool
+  default     = null
```

Comment:
Please use false

Reply:
That requires refactoring. 
Original logic assumes enforce being `true`, `false` or `empty` string
I replaced empty string with null and kept that logic (which is still used for invalid config check and creating resources)
- [local variable](https://github.com/ivankorn/terraform-google-org-policy/blob/7/main.tf#L26)
- [checks](https://github.com/ivankorn/terraform-google-org-policy/blob/7/main.tf#L29:L31)

Failed tests
```
list_constraints]$ . launch.sh
Cleaning workdir
Creating main.tf file
Test to execute: 10
 ✓ Ensure that Terraform configures the dirs and download the plugins
 ✓ Ensure that Terraform updates the plugins
 ✗ Terraform plan, ensure connection and creation of IAM binding resources
   (in test file integration.bats, line 25)
     `[[ "$output" =~ 7\ to\ add ]]' failed
 ✗ Terraform apply
   (in test file integration.bats, line 34)
     `[ "$status" -eq 0 ]' failed
 ✓ Test constraints on project gl-ivankorniienko-seed
 ✓ Test constraints on folder 147116991027
 ✗ Test constraints on organization 826592752744 (restrict values) constraining constraints/iam.allowedPolicyMemberDomains to C00u46n4k
   (in test file integration.bats, line 68)
     `[[ "$output" = "true" ]]' failed
 ✗ Test constraints on organization 826592752744 (deny values) and project gl-ivankorniienko-seed and folder 997941153340 (exclusions)
   (in test file integration.bats, line 78)
     `[[ "$output" = "true" ]]' failed
 ✗ Test constraints on folder 997941153340 (allow values)
   (in test file integration.bats, line 99)
     `[[ "$output" = "true" ]]' failed
 ✗ Terraform destroy
   (in test file integration.bats, line 110)
     `[[ "$output" =~ 7\ destroyed ]]' failed

10 tests, 6 failures
Cleaning workdir
Integration test finished
```
Example fails on config check
```
list_folder_deny]$ terraform plan
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.


------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.org-policy.null_resource.config_check[0] will be created
  + resource "null_resource" "config_check" {
      + id = (known after apply)
    }

Plan: 1 to add, 0 to change, 0 to destroy.
```